artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionBridge.java

```diff
@@ -251,29 +251,31 @@ private void setupNotificationConsumer() throws Exception {
 
          SimpleString notifQueueName = new SimpleString(qName);
 
-         SimpleString filter = new SimpleString(ManagementHelper.HDR_BINDING_TYPE + "<>" +
-                                                   BindingType.DIVERT.toInt() +
+         SimpleString filter = new SimpleString("(" + ManagementHelper.HDR_BINDING_TYPE + " <> " + BindingType.DIVERT.toInt() +
```

Comment:
Why did this existing test have to change? I would expect to not be a break change existing tests are left as was to ensure no break

Reply:
This was from #c05e1d6f26c635e3d4aea60080133f789b5f6d7c which is fixing a bug. I mentioned the semantic change in the commit message:

> Incorrect handling of unknown values in selectors.
> 
> There is a slight semantic change here due to an error in the way we
> were handling null identifiers. This may require a change in selector
> syntax to use "IS NULL" or "IS NOT NULL" when using identifiers which
> may be null in the message being selected.
> 
> This was the case for an internal filter used by the cluster connection
> bridge to select which cluster notification messages to consume.
> 
> See https://issues.apache.org/jira/browse/AMQ-5281 for more details.